creme/creme_config/forms/widgets.py

```diff
@@ -64,3 +64,49 @@ def get_context(self, name, value, attrs):
                 context = button_list.get_context(name=name, value=value, attrs=attrs)
 
         return context
+
+
+class ButtonMenuEditWidget(Widget):
+    template_name = 'creme_config/forms/widgets/buttonmenu-edit-widget.html'
+
```

Comment:
1. I try to remove the old "MyEntityEditForm" classes to replace them by "MyEntityEditionForm". So for consistency 'ButtonMenuEditionWidget' would be better.
2. Hum in 'creme_core/forms/widgets' we found XXCreatorWidget, XXXSelector, XXXEditionWidget. That's not very consistent, but the last pattern matches my remark in 1. And so ButtonMenuEditor & ButtonMenuEditorWidget could be used to. Choose one.
3. Don't forget to rename the template too ; even if you keep the 'Widget' suffix in the class, remove it from the file-name (like TimeWidget => 'widgets/time.html')